src/server.c

```diff
@@ -4165,12 +4165,6 @@ int processCommand(client *c) {
         return C_OK;
     }
 
-    /* Not allow several UNSUBSCRIBE commands executed under non-pubsub mode */
```

Comment:
Before we merge it, can we find some places that can document this behavior?

Reply:
In Valkey 7.2, these 3 commands are same as the other commands, client can call them without any restriction and there is no any special document. Thus, I think we do not need to any document for them. Let me merge this PR.

Reply:
If we want to document it, a note on the `[S|P]UNSUBSCRIBE` command pages would be a good place.

Reply:
Which kind of content we should write there? Just curious

Reply:
Example:

*In 8.0.0 and 8.0.1, this command can return an error if unsubscribe is called when not in subscribed mode, when no channels are subscribed. In 8.0.2, the behaviour from 7.2 was restored.*